From 68ebbe1603eb56f1a252a5a8db7eaf36434b2495 Mon Sep 17 00:00:00 2001
From: Kihwal Lee <kihwal@apache.org>
Date: Fri, 3 Feb 2017 15:40:34 -0600
Subject: [PATCH 2210/2848] HADOOP-14050. Add process name to kms process.
 Contributed by Rushabh S Shah.

(cherry picked from commit 7baa6dac54417d564a9233c2f76bf310bf63a822)
(cherry picked from commit 2bbcaa8ad0f50e9afe3b7945960984150c122c17)

Change-Id: I9f9eb7edc06179c67c00f9de62a3716c1d5b0aec
---
 .../hadoop-kms/src/main/sbin/kms.sh                |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-kms/src/main/sbin/kms.sh b/hadoop-common-project/hadoop-kms/src/main/sbin/kms.sh
index e24ea2e..45c14ff 100644
--- a/hadoop-common-project/hadoop-kms/src/main/sbin/kms.sh
+++ b/hadoop-common-project/hadoop-kms/src/main/sbin/kms.sh
@@ -55,7 +55,8 @@ fi
 
 print "Using   CATALINA_OPTS:       ${CATALINA_OPTS}"
 
-catalina_opts="-Dkms.home.dir=${KMS_HOME}";
+catalina_opts="-Dproc_kms"
+catalina_opts="${catalina_opts} -Dkms.home.dir=${KMS_HOME}";
 catalina_opts="${catalina_opts} -Dkms.config.dir=${KMS_CONFIG}";
 catalina_opts="${catalina_opts} -Dkms.log.dir=${KMS_LOG}";
 catalina_opts="${catalina_opts} -Dkms.temp.dir=${KMS_TEMP}";
-- 
1.7.9.5

